Bottom: 3a760e0e1523d0afc2229c36d25bdc3ae96b27e5
Top:    b280f603a328d12cd2e9bb987864cd99cdee737b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-08 23:18:15 +0100

Refresh of hid-gtk-clean-up-ghid_pan_fixu

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 36c7b7c..a4cd443 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -52,7 +52,6 @@ ghid_pan_fixup ()
   if (gport->view_width  > PCB->MaxWidth  &&
       gport->view_height > PCB->MaxHeight)
     {
-      ghid_zoom_view_fit ();
       zoom_fit ();
       return;
     }
